Documentation/eliminating-pre-builts.md

```diff
@@ -0,0 +1,85 @@
+# Eliminating pre-builts in .NET repositories
+
+This is a detailed guide on how to eliminate pre-builts in a source-buildable repository.
+It is primarily intended for developers contributing to the `dotnet` organization.
+
+## Table of content
+  - [What is a Prebuilt](#what-is-a-prebuilt)
+  - [Elimitating pre-builts](#eliminating-pre-builts)
+  - [Allowed exceptions](#allowed-exceptions)
+  - [Contacts](#contacts)
+
+# What is a Prebuilt
+
+_Source-build_ is a process of building a given product on a single machine from source with no internet access.
+
+By definition, _pre-builts_ are dependencies that a repo has on binary files that are not built from source, where _build from source_ points to any package produced during the _current source-build_. Exceptions to this rule are dependencies that are picked up from the host distro such as `cmake`.
```

Comment:
Rereading this again, it dawned on me that this statement is technically not correct - __pre-builts_ are dependencies that a repo has on binary files that are not built from source_.  A text-only nuget package would be an example of what I am referring to.  It feels more correct to say __pre-builts_ are dependencies that are not built from source.  Examples reference packages, nuget packages, and build tools._

Reply:
Agree, changed it and the next sentence to reflect this